CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Contributing
+
+Notary v2 is [Apache 2.0 licensed](https://github.com/notaryproject/nv2/blob/main/LICENSE) and
+accepts contributions via GitHub pull requests. This document outlines
+some of the conventions on to make it easier to get your contribution
+accepted.
+
+We gratefully welcome improvements to issues and documentation as well as to
+code.
+
+## Certificate of Origin
+
+By contributing to this project, you agree to the Developer Certificate of
+Origin (DCO). This document was created by the Linux Kernel community and is a
+simple statement that you, as a contributor, have the legal right to make the
+contribution.
+
+We require all commits to be signed. By signing off with your signature, you
+certify that you wrote the patch or otherwise have the right to contribute the
+material by the rules of the [DCO](https://github.com/apps/dco):
+
+`Signed-off-by: Jane Doe <jane.doe@example.com>`
+
+The signature must contain your real name
+(sorry, no pseudonyms or anonymous contributions)
+If your `user.name` and `user.email` are configured in your Git config,
+you can sign your commit automatically with `git commit -s`.
+
+As our project is about code-signing we also highly appreciate it if you sign your commits using a GPG key. :smile:
+
+## Communications
+
+For realtime communications we use Slack: To join the conversation, simply
+join the [CNCF](https://slack.cncf.io/) Slack workspace and use the
+[#notary-v2](https://cloud-native.slack.com/messages/notary-v2/) channel.
+
+To discuss ideas and specifications we use [Github
+Discussions](https://github.com/notaryproject/notaryproject/discussions).
+
+## Understanding Notary v2
+
+This project is composed of:
+
+- [notation](https://github.com/notaryproject/notation): The Notary v2 CLI and Docker plugins
+- [notation-go-lib](https://github.com/notaryproject/notation-go-lib): A collection of libraries for supporting Notation sign, verify, push, pull of oci artifacts. Based on Notary V2 standard.
```

Comment:
I know you copied this from the repo description. However, "push, pull" is out of the scope of the `notation-go-lib` and is removed now.